Linked_list/source.cpp

```diff
@@ -0,0 +1,33 @@
+#include<iostream>
+#include "linked_list.h"
+using namespace std;
+
+int main()
+{   
+    Data value,value_20;
+    value_20.a= 20;
+    List list1;
+    init(list1);
+    Node* node;
+    for (int i=0; i<10; i++)
```

Comment:
Hi @thanhhuy27021999 

Let's use the random function rand() to create your linked_list with random value.

Thanks,
NVGiang.